patches/@react-navigation+drawer+6.5.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@react-navigation/drawer/package.json b/node_modules/@react-navigation/drawer/package.json
+index e34e53f..821a033 100644
+--- a/node_modules/@react-navigation/drawer/package.json
++++ b/node_modules/@react-navigation/drawer/package.json
+@@ -65,7 +65,7 @@
+     "react": "*",
+     "react-native": "*",
+     "react-native-gesture-handler": ">= 1.0.0",
+-    "react-native-reanimated": ">= 1.0.0",
++    "react-native-reanimated": "*",
```

Comment:
Just FYI, I was curious whether the warning I linked while testing this would be fixed by upgrading `@react-navigation/drawer`, so I published a new version of our fork with this change (gitpkg and patch-package didn't seem to work together).

So you can just use `"@react-navigation/drawer": "git@github.com:Expensify/react-navigation.git#react-navigation-drawer-v6.5.0-alpha1-gitpkg"` in the package.json instead of using `patch-package` here.